main.js

```diff
@@ -0,0 +1,72 @@
+var Calculator = ()=>{
+return render=()=>{
+          let body = document.getElementsByClassName("maincontent")[0];
+          let container = document.createElement("div");
+          container.setAttribute("class","container");
+
+          let display = document.createElement("div");
+          display.setAttribute("class","display");
+
+          var displayBox = document.createElement("input");
+          displayBox.setAttribute("class","displayBar");
+          displayBox.readOnly=true;
+          display.appendChild(displayBox);
+          container.appendChild(display);
+          body.appendChild(container);
+          let numberPad = document.createElement("div");
+          numberPad.setAttribute("class","numberpad")
+
+          let buttonArray= [{title:"C",value:"C",onclick:onClear},{title:'S',value:'-1',onclick:onSign},{title:"<",value:"<",onclick:onBackspace},{title:"/",value:"/",onclick:onInput},

```

Comment:
variable name should justify the work a variable is doing. "buttonArray" can be replaced with something more specific like "calculatorKeys"